tests/test_plugin.py

```diff
@@ -107,4 +107,4 @@ def test_request_too_large(dispatcher_live_fixture):
     assert c.status_code == 200
 
     assert jdata['job_status'] == 'failed'
-    assert 'SPI-ACS backend refuses to process this request' in jdata['exit_status']['error_message']
+    assert 'SPI-ACS backend refuses to process this request' in jdata['exit_status']['message']
```

Comment:
it worries me that we expect this change. The test preserves frontend behavior. Error should go into error_message. Maybe in both.

https://github.com/oda-hub/dispatcher-app/blob/d05af3bc6dcee109f7cfd20d49632c17f867ecbc/cdci_data_analysis/flask_app/schemas.py#L12-L22

Reply:
Ok well I do not suppose the messages are shown particularly well anyway. So they can not really break.